### DISCUSSION TITLE
Why isn't there a logout endpoint in the endpoints spec?

### DISCUSSION THREAD

**Discussion by dXu23** [realworld-apps/realworld#897] [Q&A] [answered]
I've looked at the [endpoints](https://realworld-docs.netlify.app/docs/implementation-creation/introduction/), and there doesn't appear to be a logout endpoint. Should this not be a requirement?

**Comment by geromegrignon** [comment, marked as answer]
Hi @dXu23,

This project is using a client-side session rather than a server-side one.
So there is no reason to include a dedicated endpoint.
